.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -21,21 +19,17 @@ The changelogs will be integrated by the core maintainers after the merge.  See
 ### Submitter checklist
 
 - [ ] JIRA issue is well described
-- [ ] Link to JIRA ticket in description, if appropriate
+- [ ] Changelog entries provide a human-readable description of the change. [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
```

Comment:
Just some more thoughts about this:

A good example of this is #2895: Nobody *really* cares about the internal API. The motivation is the better Pipeline DSL. So the audience is users, and the message should be about Pipeline DSL.

OTOH #2899 impacts plugin developers, so they're the audience for that message (with appropriate prefix so regular users don't wonder what's going on).